tests/src/OutputNormalizerTrait.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Consolidation\OutputFormatters\Tests;
+
+/**
+ * Test Util that strips extra whitespace from
+ * the lines of a multi-line string.
+ */
+trait OutputNormalizerTrait
+{
+    /**
+     * @param $string
+     * @return string
+     */
+    protected function filterForTrailingWhitespace($string)
+    {
+        $lines = explode(PHP_EOL, $string);
+        return trim(join(" ", array_map(function ($incoming) {
```

Comment:
Any diff that may be produced would probably be easier to read if we re-join on PHP_EOL instead of space.

Reply:
rejoining them with spaces eliminates the word-wrap problem.